src/jobs/helm_aws.yml

```diff
@@ -37,14 +37,14 @@ steps:
       condition: << parameters.use_cluster>>
       steps:
         - run_aws:
-            title: Setup kubeconfig context for the cluster
+            cmd_name: Setup kubeconfig context for the cluster
             deploy: false
             cmd: |
               aws eks update-kubeconfig --name << parameters.cluster_name >>
               kubectl config get-contexts
 
         - run_aws:
-            title: Helm command - << parameters.cmd >>
+            cmd_name: Helm command - << parameters.cmd >>
```

Comment:
I would remove: `<< parameters.cmd >>`

Reply:
Seems nice to have differentiation between helm commands don't you think?